relay-replays/src/recording.rs

```diff
@@ -313,7 +313,7 @@ struct MetaEvent {
 struct CustomEvent {
     #[serde(rename = "type")]
     ty: u8,
-    timestamp: f64,
+    timestamp: Value,
```

Comment:
Would it be better to parse this into proper `Timestamp`? Like in the event protocol: https://github.com/getsentry/relay/blob/7834adb9d913b48461f7ed8f9712aa0e73f6f415/relay-general/src/protocol/event.rs#L331

Reply:
Or it can be just `DataTime` that `serde` supports